packages/next/package.json

```diff
@@ -84,7 +84,7 @@
     "loader-utils": "2.0.0",
     "native-url": "0.3.4",
     "node-fetch": "2.6.1",
-    "node-html-parser": "^1.2.19",
+    "node-html-parser": "1.4.9",
```

Comment:
went ahead and updated this per @devknoll suggestion, but it looks like one of the versions fixed some string serialization issue with `styles`. May as well 🤷

Reply:
I'm thinking we should do this in another PR